linera-chain/src/manager.rs

```diff
@@ -455,6 +453,7 @@ impl ChainManager {
         validated: ValidatedBlockCertificate,
         key_pair: Option<&KeyPair>,
         local_time: Timestamp,
+        blobs: BTreeMap<BlobId, Blob>,
```

Comment:
Is it correct to say that all blobs that are either published or read - for the first time on this chain - should be in `blobs` and all blob from `blobs` should be one of the former? i.e. basically that `blobs.keys === valitated.required_blob_ids`

Reply:
Yes, that's true. I'll add a comment to the method.

Reply:
I was thinking about making it a `debug_assert!` bo do we even have that in our compiled (and ran) binaries? 🤔

Reply:
We also have some `assert!`s in the production code, I think, but I'm always scared to add more. :sweat_smile: